pipeline_dp/accumulator.py

```diff
@@ -0,0 +1,82 @@
+import abc
+import typing
+
+
+class Accumulator(abc.ABC):
+  """
+  Performs aggregations
+  """
+
+  @classmethod
+  def merge(cls, accumulators: typing.Iterable[
+    'Accumulator'] = None) -> 'Accumulator':
```

Comment:
How do you like the idea of args type argument here.
```
merge(cls, *accumulators: 'Accumulator' = None) 
```

A list need not be created when a single accumulator is involved.
But when a list is involved, then they have to unpack it merge(*list_of_acc)

May not be super helpful, but wanted to know your thoughts on this.

Reply:
Got it! Thanks!